src/components/analytics/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unused-modules */
```

Comment:
I feel like this `eslint` rule is causing more issues than solutions. Should be just warning, not a full-blown error

Reply:
Good catch, this should actually be removed, I only added before when I was integrating bc it was annoyingly breaking my build while I was not using it yet, will remove it now 👍